core/index.ts

```diff
@@ -0,0 +1,2 @@
+export { User } from './auth'
+export { Arrays, Numbers, Objects, UUIDs } from './utils'
```

Comment:
no need to export everything from index.
e.g. user is imported from core/auth etc..